tools/src/main/java/org/apache/kafka/tools/TopicCommand.java

```diff
@@ -469,7 +469,7 @@ public void createTopic(CommandTopicPartition topic) throws Exception {
                 CreateTopicsResult createResult = adminClient.createTopics(Collections.singleton(newTopic),
                     new CreateTopicsOptions().retryOnQuotaViolation(false));
                 createResult.all().get();
-                System.out.println("Created topic " + topic.name + ".");
+                System.out.println("Created topic " + topic.name.get() + ".");
```

Comment:
Thanks for the fix. I think we should pass the topic name to `createTopic` so we don't have to call `.get` in so many places.

Reply:
Thanks for reviewing! 

Instead of adding topicName argument to `createTopic`, could we change the type of `CommandTopicPartition.name` from `Optional<String>` to `String`?
`CommandTopicPartition` class is used in `createTopic` and `alterTopic`, and these operations require `--topic` option. Therefore, it does not seem necessary to be `Optional`.

Or, since we already have the topic name,  we could simply define a local variable `String topicName = topic.name.get()` in `createTopic` to avoid calling `.get` method multiple times.

What do you think about this?

Reply:
If it doesn't have to be `Optional`, it's even better. Worth trying and seeing if you run into any issues.

Reply:
I changed the type of it. And it seems to be working.